test/system/710-start.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats   -*- bats -*-
```

Comment:
Would you consider renaming to `045-start.bats`? The purpose of the naming scheme isn't "keep appending to the end", it's "what is important?" In this case, `start` is much much more important than networking, completion, play, and so on. `45` seems like a good place for it, right before `stop`. The fact that there wasn't already a `start` file is purely an oversight on my part.